setup.py

```diff
@@ -2,7 +2,7 @@
 
 if __name__ == '__main__':
     setup(
-        version='1.3.3',
+        version='1.3.4',
```

Comment:
semver - new feature that _might_ affect some users
```suggestion
        version='1.4.0',
```